### ISSUE TITLE
Call of duty mobile KMT STEER WHEEL problem

### ISSUE THREAD

**Issue by botakkeren** [barry-ran/QtScrcpy#278]
I have a little problem with steer wheel problem on Call of duty, i dont know how to describe it, but the control is pretty mess up. When try my all of duty  streer wheel control setting  in to PUBG, it works good no problem at all.

**Comment by hugonh** [comment]
me too, exactly!!!

**Comment by mrsoseji** [comment]
Exactly. The WASD works fairly okay on PUBGM but not on CODM. I've tried many variations with different values and different ingame settings but it just wouldn't work properly. Others work great! Hope they would get it fixed soon.

**Comment by jirehnimes** [comment]
Same issue. Even when the keymap rules are correct. Like when moving it will steer the screen instead. Sometimes when pressing numbers it will move away the cursor. Hope it will be fixed soon for CODM.

**Comment by hugonh** [comment]
not just cod, i tried others games too, black desert online, with all possible configs

**Comment by EgyptianM** [comment]
Anyone has a keymap config file for CODM?

**Comment by EgyptianM** [comment]
It's CODM issue
I tried many other software but it's the same mess 
Tried Apowersoft mirror the premium version 
It happens even when I'm playing from phone directly with static joystick

**Comment by barry-ran** [comment]
try [v1.7.0](https://github.com/barry-ran/QtScrcpy/releases/tag/v1.7.0)

**Comment by TheSatanas** [comment]
Anyone have the keymap file for codm? 
I dont know how to code..
